File: lab1.java
class MyThread extends Thread {
    public static void main(String[] args) {
        Thread t = new Thread(() -> System.out.println("Hello from a thread!"));
 
        t.start();
    }
}


class lab1 {
    public static void main(String[] args) {
        for (int i = 0; i < 10; i++) {
            System.out.println("Hello from main!");
            Thread t = new Thread(() -> System.out.println("Hello from a thread!"));
            t.start();
        }
    }
}

File: lab1.kt
internal object MyThread : Thread() {
    fun main(args: Array<String?>?) {
        val t = Thread { System.out.println("Hello from a thread!") }
        t.start()
    }
}

internal object lab1 {
    fun main(args: Array<String?>?) {
        for (i in 0..9) {
            System.out.println("Hello from main!")
            val t = Thread { System.out.println("Hello from a thread!") }
            t.start()
        }
    }
}